miden/tests/integration/helpers/mod.rs

```diff
@@ -205,13 +202,14 @@ impl Test {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-/// Takes an array of u64 values and builds a stack, preserving their order and converting them to
-/// field elements.
-pub fn convert_to_stack(values: &[u64]) -> [Felt; STACK_TOP_SIZE] {
-    let mut result = [Felt::ZERO; STACK_TOP_SIZE];
-    for (&value, result) in values.iter().zip(result.iter_mut()) {
-        *result = Felt::new(value);
-    }
+/// Converts an array of Felts into u64
+pub fn stack_to_int(values: &[Felt]) -> Vec<u64> {
+    values.iter().map(|e| (*e).as_int()).collect()
+}
+
+pub fn stack_to_size(values: &[u64]) -> Vec<u64> {
+    let mut result: Vec<u64> = values.iter().cloned().collect();
+    result.resize(STACK_TOP_SIZE, 0);
```

Comment:
this will truncate the data, if the user gave more than `STACK_TOP_SIZE` elements, the FRI tests rely on that behavior.